### PR TITLE
Correct draw.aaline docs

### PR DESCRIPTION
**Pull request by Starbuck5**
I see this is addressed already in #3191, but I'd like to get it deployed without waiting on the rest of the PR to be reviewed. To keep on docs on the website as correct as possible.

### CODE REVIEW

File: docs/reST/ref/draw.rst [hunk 1]
```diff
@@ -493,7 +493,7 @@ object around the draw calls (see :func:`pygame.Surface.lock` and
    .. versionchangedold:: 2.0.0 Added support for keyword arguments.
    .. versionchanged:: 2.4.0 Removed deprecated 'blend' argument
    .. versionchanged:: 2.5.0 ``blend`` argument readded for backcompat, but will always raise a deprecation exception when used
-   .. versionchanged:: 2.5.2 Added line width
+   .. versionchanged:: 2.5.3 Added line width
```
Comment (ankith26) [inline docs/reST/ref/draw.rst:496]: There are more references to `width` in this block, though

Comment (Starbuck5) [reply docs/reST/ref/draw.rst:496]: Yes, and that's fine. The docs on the site always have the features of main.

**Review by ankith26 — APPROVED** [review]
Yeah, lets get this in. Thanks! :tada: 

This PR got me thinking if it's better to stick to latest-stable-release docs on the website. I think that would be less confusing for users and prevent this kind of situation entirely. What say?

**Comment by Starbuck5** [comment]
Well if we're going to be fancy we could have a version selector in our docs, like Python does.

**Comment by Starbuck5** [comment]
But I guess that would be fine? Prerelease and testing users could use local docs?

**Review by oddbookworm — APPROVED** [review]
LGTM!

**Pull request merged**